BossMod/Autorotation/RotationModule.cs

```diff
@@ -23,10 +23,12 @@ public enum RotationModuleQuality
 // the configuration part of the rotation module
 // importantly, it defines constraints (supported classes and level ranges) and strategy configs (with their sets of possible options) used by the module to make its decisions
 // rotation modules can optionally be constrained to a specific boss module, if they are used to implement custom encounter-specific logic - these would only be available in plans for that module
-public sealed record class RotationModuleDefinition(string DisplayName, string Description, string Author, RotationModuleQuality Quality, BitMask Classes, int MaxLevel, int MinLevel = 1, Type? RelatedBossModule = null)
+public sealed record class RotationModuleDefinition(string DisplayName, string Description, string Author, RotationModuleQuality Quality, BitMask Classes, int MaxLevel, int MinLevel = 1, Type? RelatedBossModule = null, bool CanUseWhileMounted = false, bool CanUseWhileRoleplaying = false)
```

Comment:
actually if all the RP stuff is going to be boss module specific, then no, these flags are pointless

Reply:
wait no, isn't there a vehicle mechanic in Rival Wings? also, Praetorium has the magitek armor section, and i'm not sure how a module would work for that, since there's not really a Boss and modules require one to be defined

Reply:
I kinda don't see people including specific rotation module in their presets just for prae?

And module could still be defined for a trash I guess... Well, I think i'll kill it for now, and if we end up needing it later, we'll remake it.

Reply:
i was thinking more of one preset for "all vehicle combat" that would get included in the default presets list. would be useful not only for prae but also for the "magiteknical failure" aether current sidequest in Stormblood. and also wherever else vehicle combat is used, which is basically nowhere, but it's so simple it doesn't seem like that big of a deal to implement